lib/Doctrine/ruleset.xml

```diff
@@ -100,12 +100,16 @@
                 name="forbiddenAnnotations"
                 type="array"
                 value="
+                    @api,
                     @author,
+                    @category,
                     @copyright,
                     @created,
                     @license,
                     @package,
                     @since,
+                    @subpackage,
+                    @uses,
```

Comment:
`@uses` is complimentary with `@covers` in test suites, and affects phpunit at runtime when using strict coverage checks

Reply:
Oh, I see it https://phpunit.de/manual/current/en/appendixes.annotations.html, sorry!